### PR TITLE
Troubleshooting mutliple leecher test

### PR DESCRIPTION
**Pull request by tomclose**
Fixes https://github.com/statechannels/monorepo/issues/2215

### CODE REVIEW

File: packages/web3torrent/src/library/web3torrent-lib.ts [hunk 1]
```diff
@@ -582,6 +583,26 @@ export default class WebTorrentPaidStreamingClient extends WebTorrent {
     );
   }
 
+  protected async waitForMySeederChannelsToClose(torrent: PaidStreamingTorrent) {
```
Comment (geoknee) [inline packages/web3torrent/src/library/web3torrent-lib.ts:586]: are MySeederChannels the same as mySeedingChannels?

**Comment by andrewgordstewart** [comment]
These are the screenshots at the end of the multiple-leecher test on 42eb44e.

Screen Shot 2020-06-17 at 9 29 29 AM

I think React is still reacting to information, because the `.html` files are a bit different.

I am confused about who this is:
Screen Shot 2020-06-17 at 9 49 55 AM

**Comment by geoknee** [comment]
@andrewgordstewart I don't think this branch includes this change https://github.com/statechannels/monorepo/pull/2216, which solved a problem about inconsistent blockies.

**Comment by andrewgordstewart** [comment]
> I am confused about who this is:
> Screen Shot 2020-06-17 at 9 49 55 AM

Actually, none of the _peers_ blockies match up with any of the _who am I_ blockies.

**Pull request merged**